### ISSUE TITLE
Resume doesn't work if bot has been paused for a long time

### ISSUE THREAD

**Issue by rafaeldamasceno** [galnir/Master-Bot#579]
**Describe the bug**
If you pause the bot and let it sit there for a long time (5~10 minutes), when you try to resume the song, it doesn't play anything.

**To reproduce**
1. Queue and play any song
1. Pause using the button or the pause command
1. Do nothing for some time
1. Resume using the button or the resume comand

**Expected behavior**
Bot resumes playing the song from the time it was paused.

**Actual behavior**
Bot replies that it resumed playing, but no sound is played. Using the now playing command shows the bot frozen at the pause timestamp.

**Desktop (please complete the following information):**
 - OS: Raspbian GNU/Linux 10 (buster), armhf
 - How are you hosting the bot: Docker using the image and container from the existing Dockerfile

**Additional context**
 Skipping the current song gets the bot working again.

**Comment by galnir** [comment]
Hey @rafaeldamasceno, sorry for the late response.
I failed to replicate this issue. I paused the music for 8 minutes then hit resume and the music was playing again instantly.
@Bacon-Fixation can you try to replicate this?

**Comment by galnir** [comment]
@rafaeldamasceno It's actually working for me now for some reason, is it still not resuming for you?

Also related:
@Bacon-Fixation Is there a reason for disabling the reaction buttons after 10 minutes?

**Comment by Bacon-Fixation** [comment]
It was so it didnt leave reaction open when left in a pause state

**Comment by galnir** [comment]
> It was so it didnt leave reaction open when left in a pause state

Are you able to replicate this issue?

**Comment by Bacon-Fixation** [comment]
nah, i haven't been able to.

**Comment by galnir** [comment]
@rafaeldamasceno Are you still experiencing this?

**Comment by rafaeldamasceno** [comment]
Yes, I still have the issue. I put a timer for 12 minutes and resume didn't work. 

https://user-images.githubusercontent.com/6492427/125069052-23885280-e0ae-11eb-8585-a67708b67b68.mov

When trying to record the video, I accidentally paused and resumed again and *then* it worked, albeit with a weird issue where it skipped for all the time it should have been playing after the first resume but didn't.

**Comment by galnir** [comment]
@rafaeldamasceno Did you try to resume using the resume command or the resume emoji?
Also were you able to use other music commands when it happened?

**Comment by rafaeldamasceno** [comment]
I always used the message commands. Here's a workflow of how I reproduced it and recorded it on the following video:

1. Play a song.
2. Pause the song and wait 5 minutes. Use np to check timestamp (1:07).
3. Resume the song. Nothing happens. Wait for a little bit and check timestamp with np (1:07). It's the same as if the song hasn't been resumed.
4. What I have discovered today is that if you first pause again, and then resume, the song will actually resume, but play fast/skip for all the time since you have first resumed in step 3.

https://user-images.githubusercontent.com/6492427/125111453-d4f2ac80-e0dd-11eb-85b5-8440740d26f9.mov

All of the other commands, like skip, stop, etc. work. It's just the current track that gets stuck.

**Comment by galnir** [comment]
@rafaeldamasceno 
I tested this in different Node versions and came to a conclusion that newer Node 14 versions break the resume command. I suggest you to downgrade your Node version to v14.15.4 (the one I'm using), and after discord.js releases v13 and their new discord.js/voice library I will make sure to use them to solve this
